Stubs/Matcher.astub

```diff
@@ -0,0 +1,39 @@
+import org.checkerframework.common.value.qual.MinLen;
```

Comment:
For this (and other stub files), I'd suggest removing methods you're not annotating - it makes it much more clear what the assumptions you're making are.